source/standards/continuous-delivery.html.md.erb

```diff
@@ -1,42 +1,39 @@
 ---
 title: Use continuous delivery
-last_reviewed_on: 2020-07-06
+last_reviewed_on: 2021-03-18
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
 [Continuous delivery][] helps you implement changes like new features into production quickly and safely. 
 
+## Benefits
+
 Using continuous delivery has multiple benefits.
 
-## Iterate code frequently
+### Iterate code frequently
 
 It’s cheaper and easier to deliver small changes to production, meaning you can get feedback from users quickly.
 
-## Low-risk releases
+### Low-risk releases
 
 Frequent small changes make it easier to diagnose problems, and the cost of fixing them is much lower than with large releases.
 
-## Quality software builds
+### Quality software builds
 
 Automated test suites allow you to quickly identify regressions in your software. This means you can focus on exploratory, usability, performance and security testing.
 
-## Maintainable code
+### Maintainable code
 
 Building and releasing software in small pieces helps you focus on writing small composable bits of code. This means your code is easier to maintain and update.
 
 ## Essential concepts
 
-With continuous delivery use frequent integration with the main branch, automatic build promotion and production monitoring.
-
-### Frequent integrations with main branch
-
-With [continuous integration][] you integrate with a main branch at least once a day, for example using source-control branching models like [Trunk-Based Development][].
+To achieve continuous delivery you need:
 
-If your team’s environment is regulated, and you need evidence that you review all code changes, raise and merge frequent small pull requests. For example, [Fourth Wall Helpful][], a client-side pull request and build status monitor for Github repositories, could help you build a team culture to support this practice.
-
-Use approaches like [feature-flagging][] or [modular architectures][] to deploy partially complete features. This reduces the size of changes going to production and encourages your team to build modular, configurable systems. 
```

Comment:
I quite liked this, too. Feature-flagging goes alongside A/B testing in a lot of cases, so maybe we could link off to something about that.

Reply:
I've added this back in